apps/pcq/pcq-backend/demo-image-policy.yaml

```diff
@@ -2,6 +2,14 @@ apiVersion: image.toolkit.fluxcd.io/v1beta2
 kind: ImagePolicy
 metadata:
   name: demo-pcq-backend
+  annotations:
```

Comment:
this is not compelete you have the following options
1. update demo.yaml to point to this image policy _demo-pcq-backend_ (there are many examples)
2. Point directly from demo.yaml to your pr image (see perftest.yaml in ccd-case-disposer)